src/EPR.Calculator.Frontend/Views/ParameterConfirmation/Index.cshtml

```diff
@@ -24,4 +29,11 @@
             </div>
         </div>
     </main>
-</div>
\ No newline at end of file
+</div>
+<script type="text/javascript">
+    $(document).ready(() => {
+        var dt = new Date();
```

Comment:
Please use `const `instead of `var`

Reply:
`const event = new Date(Date.UTC(2012, 11, 20, 3, 0, 0));
const options = {
  weekday: 'long',
  year: 'numeric',
  month: 'long',
  day: 'numeric',
};`

Reply:
I have now updated the code to use const instead of var.